src/pybind/mgr/dashboard/frontend/src/styles/ceph-custom/_basics.scss

```diff
@@ -35,12 +35,6 @@ option {
   justify-content: center;
 }
 
-.loading {
```

Comment:
@nizamial09 can you please check if this has any impact on branding?

Reply:
@alfonsomthd Yup. I'll test it today along with the branding impact.